src/jobs/publish_s3.yml

```diff
@@ -0,0 +1,35 @@
+description: Publish files to S3
+
+executor: aws
+
+parameters:
+  public:
+    type: boolean
+    description: Whether to store in the public bucket
+  path:
+    type: string
+    description: Path to store the files in S3
+  files:
+    type: string
+    description: Files to publish separated by space
+
+steps:
+  - attach_workspace:
+      at: ~/workdir
+
+  - run_aws:
+      cmd_name: Send files to S3
+      cmd: |
+        if [[ << parameters.public >> ]]; then
+          bucket=$AWS_PUBLIC_ARTIFACTS_BUCKET
+        else
+          bucket=$AWS_PRIVATE_ARTIFACTS_BUCKET
+        fi
+
+        for file in << parameters.files >> ; do
```

Comment:
does this work? :O where do we specify that they are comma separated? how is this already an array?

Reply:
Yeah, @h314to Bash magic :-D